hack/tools/Makefile

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export GO111MODULE=on
+
+GOBIN ?= $(shell go env GOPATH)/bin
```

Comment:
probably we should prefer a local binary dir instead of clobbering the system binaries?

Reply:
We don't use this from our build harness it can be used by folks who want to work offline, we could make it clearer in the README that they can override the GOBIN to place it elsewhere

Reply:
ack